doc/extension_store/manifest.jsonc

```diff
@@ -0,0 +1,139 @@
+{
+    // Info displayed in the Extension Store
+    "id": "postgres-vs",
+    "name": "PostgreSQL Virtual Schema",
+    "description": "...",
```

Comment:
Add a version of the manifest. In the future we could want to change the format. Then we can't change all repos at once.